src/test/java/uk/gov/ons/ctp/integration/envoycuc/client/RateLimiterClientWebformRequest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ons.ctp.integration.envoycuc.client;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.ons.ctp.integration.ratelimiter.client.RateLimiterClient.Domain;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class RateLimiterClientWebformRequest extends RateLimiterClientRequest {
```

Comment:
Why extend RateLimiterClientRequest? 
webform limiting only uses the domain and ipAddress.

Reply:
because I call methods in my MockClient using the base class so I don't have to duplicate the code. 
The rate limiter itself is called with only the fields it needs

Reply:
ok. I can see how you want to be able to write code which works with a RateLimiterClientRequest. 
In which case the RateLimiterClientRequest should only contain fields which are common to all limit requests, with the subclasses containing the request specific fields.
If you don't already have it then it looks like you need a RateLimiterClientFulfilmentRequest class.

Reply:
OK - I have removed those classes from the MockLimiter and I only use the two classes in the  Steps class to store values in a List object before I post